README.md

```diff
@@ -148,16 +148,15 @@ This section lists each configuration option, and whether it can be set by each
 | Facility |    | `METAL_FACILITY_NAME` | `facility` | read metadata on host on which CCM is running, else error |
 | Base URL to Equinix API |    |    | `base-url` | Official Equinix Metal API |
 | Load balancer setting |   | `METAL_LB` | `loadbalancer` | none |
-| BGP ASN for each cluster node |   | `METAL_LOCAL_ASN` | `localASN` | `65000` |
-| BGP ASN for upstream peer |   | `METAL_PEER_ASN` | `peerASN` | `65530` |
+| BGP ASN for cluster nodes when enabling BGP on the project |   | `METAL_LOCAL_ASN` | `localASN` | `65000` |
+| BGP passphrase to use when enabling BGP on the project |   | `METAL_BGP_PASS` | `bgpPass` | `""` |
 | Kubernetes annotation to set node's BGP ASN |   | `METAL_ANNOTATION_LOCAL_ASN` | `annotationLocalASN` | `"metal.equinix.com/node-asn"` |
 | Kubernetes annotation to set BGP peer's ASN |   | `METAL_ANNOTATION_PEER_ASNS` | `annotationPeerASNs` | `"metal.equinix.com/peer-asn"` |
 | Kubernetes annotation to set BGP peer's IPs |   | `METAL_ANNOTATION_PEER_IPS` | `annotationPeerIPs` | `"metal.equinix.com/peer-ip"` |
 | Kubernetes annotation to set source IP for BGP peering |   | `METAL_ANNOTATION_SRC_IP` | `annotationSrcIP` | `"metal.equinix.com/src-ip"` |
-| Kubernetes annotation to set source IP for BGP peering |   | `METAL_ANNOTATION_SRC_IP` | `annotationSrcIP` | `"metal.equinix.com/src-ip"` |
+| Kubernetes annotation to set BGP MD5 password, base64-encoded |   | `METAL_ANNOTATION_BGP_PASS` | `annotationBGPPass` | `"metal.equinix.com/bgp-pass"` |
```

Comment:
Can we add a footnote here, or otherwise add a warning to this doc around exposing this via an annotation on the node to pre-preemptively address concerns folks may have?

Reply:
Definitely a good call. Adding.

Reply:
Take a look now, @detiber

Reply:
lgtm, thanks!